content/en/synthetics/browser_tests/_index.md

```diff
@@ -35,23 +35,27 @@ Browser tests are scenarios executed by Datadog on your web applications. They r
 
 Define the configuration of your browser test.
 
-1. Enter a **Starting URL**: The URL from which your browser test starts the scenario.
+1. Select a pre-populated template to choose from, or to build a test from scratch see steps 2-7 below.
+
+   {{< img src="getting_started/synthetics/synthetics_browser_templates.png" alt="Synthetics browser test landing page with templates" style="width:100%;" >}}
+
+2. If you want to build a test from scratch, click the **+** template, then enter a **Starting URL**: The URL from which your browser test starts the scenario.
```

Comment:
Same comment as above; it's nice when the structure of a list can help clue you in on exactly which steps are mutually exclusive in a procedure, especially if they converge again later.